tests/PHPStan/Parser/PhpDocParserTest.php

```diff
@@ -5526,6 +5562,98 @@ public function provideSelfOutTagsData(): Iterator
 		];
 	}
 
+	public function provideCommentLikeDescriptions(): Iterator
+	{
+		yield [
+			'Comment after @param',
+			'/** @param int $a // this is a description */',
```

Comment:
It is *not* a description, it is a _commented_ description, meaning that as far as the parser is concerned there is no description here.